components/src/molecules/ParametersTable/InfoScreen.tsx

```diff
@@ -54,30 +54,55 @@ export function InfoScreen({
           ? t('no_offsets_available')
           : 'No Labware Offset data available'
       break
+    case 'noLabwareOffsetDataYet':
+      bodyText =
+        t != null ? t('no_labware_offset_data') : 'No labware offset data yet'
```

Comment:
because of `<InfoScreen contentType="noLabwareOffsetDataYet" />`

```suggestion
      bodyText = 'No labware offset data yet'
```

I will refactor this component and components that use this component to remove `t` since I believe that consuming i18n instance in a component wouldn't good.

Reply:
Sounds good

Reply:
created the ticket for this.
https://opentrons.atlassian.net/browse/AUTH-746